src/resources/locales/index.ts

```diff
@@ -18,5 +19,6 @@ export default {
   "zh-TW": zhTW,
   "zh-CN": zhCN,
   et,
+  "pl-PL": pl,
```

Comment:
```suggestion
  pl,
```
spotify is confusing me